recipes/morphotactics/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "morphotactics" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dmort27/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0c0ff9fc35acb9281438e4412f40c46f515cb1f0998a0ab301a0d1ce0c2a07bc
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win or py<36]
```

I'm pretty sure this can be made noarch (and we can ignore the Windows build failure--noarch packages once they're made into feedstocks are only built on Linux).
```suggestion
  noarch: python
```